### ISSUE TITLE
Browse page and Show Jobs page broken in IE 11

### ISSUE THREAD

**Issue by W-Park** [fossology/fossology#1095]
Browse page and Show Jobs page are broken in IE 11 even though they works fine in other browsers.
Is there any suggestions to fix this?

Chrome 66.0.3359.139
![image](https://user-images.githubusercontent.com/33045921/39792041-ae75b448-5379-11e8-820b-570897ed365c.png)
![image](https://user-images.githubusercontent.com/33045921/39792072-e077efb0-5379-11e8-815c-1bd64d651152.png)

IE 11.0.9600.18920
![image](https://user-images.githubusercontent.com/33045921/39792037-a71c6354-5379-11e8-8639-0bac01cffe6f.png)
![image](https://user-images.githubusercontent.com/33045921/39792070-dc2e0d5e-5379-11e8-91da-93979b71c6bf.png)

**Comment by mcjaeger** [comment]
This is right, at some point we saw that maintaining IE11 is maybe difficult. I also know that it is not working. IE11 does not seem to work properly with basic Ui frameworks such as jquery. And we can decide whether to have features or IE11 support.

Please refer to the histoery of the IE11 application:
https://en.wikipedia.org/wiki/Internet_Explorer_11
https://en.wikipedia.org/wiki/Internet_Explorer

**Comment by mcjaeger** [comment]
I was trying to understand more of its root cause. Maybe a short fix could help here:

1. up to my knowledge, the jquery UI elements do not work in fossology even with "compatibility mode" switched off in IE11. You can try it, but AFAIK it does not help here.

2. I heave found this and maybe this is page metedata addition is useful: https://stackoverflow.com/questions/25077612/jquery-not-working-with-ie-11

I am still not convinced to care much for IE11 which is subject for being replaced by edge.

**Comment by W-Park** [comment]
Thank you for your kind reply.
I understand your point.
It could be a burdensome that people are not gonna use IE11 anymore.

I suppose I have to advise the users to use other then IE11.

Shell I close this issue?

**Comment by W-Park** [comment]
I suppose it was related to intranet issue. (https://stackoverflow.com/questions/30132573/jquery-is-not-loading-when-i-add-intranet-site-to-intranet-security-zone-ie11)
My IE was changing setting to IE7 when loading the fossology service (by default).
And when I changed it to IE9, IE10 or edge at developer tool, it worked fine.

So I simply added followings and it worked for me.

- /usr/local/share/fossology/lib/php/Plugin/Default_Plugin.php
-- 358 $metadata .= "<meta http-equiv='X-UA-Compatible' content='IE=edge'>\n";
- /usr/local/share/fossology/lib/php/Plugin/FO_Plugin.php
-- 334     $metadata .= "<meta http-equiv='X-UA-Compatible' content='IE=edge'>\n";

Thank you mcjaeger for your help!!

**Comment by mcjaeger** [comment]
actually, it maybe useful (if not interfering), but maybe it should be put into the main master branch of fossology.

**Comment by GMishx** [comment]
It would be better if we add the meta tags in `src/www/ui/template/include/head.html.twig` as well. Because every page rendered using twig include this file.

**Comment by W-Park** [comment]
Regretfully, I am not really experienced on web development, so I can't tell if it interferes with other codes or not. (It seems there is no problem occurred  so far.)
But if it goes into master branch, there would be one less job for me on next update :)

**Comment by shaheemazmalmmd** [comment]
Hello,

This issue has been inactive for several years and appears to no longer be actionable. As part of our issue tracker cleanup, we are closing old inactive issues.

If you believe this is still a valid issue in the current version of FOSSology, please comment with updated information or open a new issue with reproduction steps, and we'll be glad to investigate.

Thank you for your understanding and your support of the project!
